testsuite/MDAnalysisTests/lib/test_pkdtree.py

```diff
@@ -129,19 +138,19 @@ def test_ckd_searchpairs_nopbc(radius, result):
     assert_equal(indices, result)
 
 
-@pytest.mark.parametrize('b, q, result', (
-                         ([10, 10, 10, 90, 90, 90], [0.5, -0.1, 1.1], []),
-                         ([10, 10, 10, 90, 90, 90], [2.1, -3.1, 0.1], [[0, 2],
-                                                                       [0, 3],
-                                                                       [0, 4]]),
-                         ([10, 10, 10, 90, 90, 90], [[2.1, -3.1, 0.1],
-                                                     [0.5, 0.5, 0.5]], [[0, 2],
-                                                                        [0, 3],
-                                                                        [0, 4],
-                                                                        [1, 1]]),
-                         ([10, 10, 10, 45, 60, 90], [2.1, -3.1, 0.1], [[0, 2],
-                                                                       [0, 3]])
-                         ))
+@pytest.mark.parametrize(
+    "b, q, result",
+    (
+        ([10, 10, 10, 90, 90, 90], [0.5, -0.1, 1.1], []),
```

Comment:
it feels like the "old" way was intentionally formatted that way, but this might still be more readable

Reply:
Which one should we keep? I think the new one is alright, but I don't mind either way.

Reply:
I don't see any meaningful difference between them, so I'd keep the new one for consistency.

Reply:
I think the old formatting was supposed to highlight the shape of the array visually. I don't mind either way.

Reply:
Yeah exactly, by visually highlighting the shape is makes is easier to "think through" the problem you're trying to test for, rather than doing it with flat array representations.

I'm happy with either, but possibly a slight preference for the old style (in the sense that it keeps whatever pedagogic value it used to have).

Reply:
Ok, I kept the old explicit formatting on this one.